client/state/help/ticket/test/actions.js

```diff
@@ -72,17 +73,19 @@ describe( 'ticket-support/configuration actions', () => {
 		test( 'should be failed.', () => {
 			const action = ticketSupportConfigurationRequest()( spy );
 
-			assert( spy.calledWith( sinon.match( { type: HELP_TICKET_CONFIGURATION_REQUEST } ) ) );
+			expect(
+				spy.calledWith( sinon.match( { type: HELP_TICKET_CONFIGURATION_REQUEST } ) )
+			).toBeTruthy();
 
-			action.then( () => {
-				assert(
+			return action.then( () => {
```

Comment:
Why do we need to return the action promise in these cases?

Reply:
Jest requires that we either return or await promises to be resolved when doing assertions on their result. See https://jestjs.io/docs/tutorial-async and [this particular ESLint rule](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-expect-in-promise.md).